src/schema/v2/artist/index.ts

```diff
@@ -80,11 +80,23 @@ export const artistArtworkArrayLength = (artist, filter) => {
 
 export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
   name: "Artist",
-  interfaces: [NodeInterface, Searchable],
+  interfaces: () => {
```

Comment:
thunk all the things!

Reply:
Oh lol I was just joking about thunking everything in the sense that,`fields` were what usually get thunked but here `interfaces` are getting thunk'ed, and not that you had to thunk all the interfaces here!

Sorry for not making that clearer...was just joking! 😅